### PR TITLE
Adjust visuals sync progress

### PR DESCRIPTION
**Pull request by Rahmon**
### Checklist:

- [ ] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: assets/js/sync.js [hunk 1]
```diff
@@ -1,23 +1,178 @@
...
+
+function deleteAndSync() {
+	activeBox = deleteAndSyncBox;
+
+	disableButtonsInSyncBox();
+	updateDisabledAttribute(deleteAndSyncButton, true);
+	showPauseStopButtons();
+	showProgress();
+
+	addLineToOutput(__('Deleting all data…', 'elasticpress'));
```
Comment (felipeelia) [inline assets/js/sync.js:78]: This isn't accurate (or shouldn't be), as we are not deleting everything at once.

File: includes/classes/IndexHelper.php [hunk 2]
```diff
@@ -93,14 +93,17 @@ protected function build_index_meta() {
...
-			'network_alias' => [],
-			'start_time'    => microtime( true ),
-			'totals'        => [
+			'method'          => ! empty( $this->args['method'] ) ? $this->args['method'] : 'web',
+			'offset'          => ! empty( $this->args['offset'] ) ? absint( $this->args['offset'] ) : 0,
+			'start'           => true,
+			'sync_stack'      => [],
+			'network_alias'   => [],
+			'start_time'      => microtime( true ),
+			'start_date_time' => $start_date_time ? $start_date_time->format( 'D, F d, Y H:i' ) : false,
```
Comment (felipeelia) [inline includes/classes/IndexHelper.php:105]: Not a blocker for Beta 1 but we should have this stored in another format, so people can translate that format later.

File: includes/partials/sync-page.php [hunk 3]
```diff
@@ -2,35 +2,209 @@
...
+						</button>
+					</div>
+				</div>
+			</div>
+			<div class="ep-sync-box__progress-wrapper">
+				<div class="ep-sync-box__progress">
+					<div class="ep-sync-box__sync-in-progress">
+						<img width="36" height="36" src="<?php echo esc_url( plugins_url( '/images/sync-in-progress.png', dirname( __DIR__ ) ) ); ?>" />
+						<div class="ep-sync-box__sync-in-progress-info">
+							<div class="ep-sync-box__progress-info">Sync in progress</div>
```
Comment (felipeelia) [inline includes/partials/sync-page.php:85]: can we wrap this text with a esc_html__() call?

File: includes/partials/sync-page.php [hunk 4]
```diff
@@ -2,35 +2,209 @@
...
+						</span>
+					</button>
+				</div>
+			</div>
+			<div class="ep-sync-box__progress-wrapper">
+				<div class="ep-sync-box__progress">
+					<div class="ep-sync-box__sync-in-progress">
+						<img width="36" height="36" src="<?php echo esc_url( plugins_url( '/images/sync-in-progress.png', dirname( __DIR__ ) ) ); ?>" />
+						<div class="ep-sync-box__sync-in-progress-info">
+							<div class="ep-sync-box__progress-info">Sync in progress</div>
```
Comment (felipeelia) [inline includes/partials/sync-page.php:166]: same as above

File: includes/partials/sync-page.php [hunk 5]
```diff
@@ -2,35 +2,209 @@
...
+							<div class="ep-sync-box__start-time">Start time: <span class="ep-sync-box__start-time-date">Fri, September 30, 2021 16:20</span></div>
```
Comment (felipeelia) [inline includes/partials/sync-page.php:167]: this one too :)

**Review by felipeelia — CHANGES_REQUESTED** [review]
Some issues I found during my tests:

### General
- [x] Progress bar doesn't seem to be working. As soon as I click on Sync Now, it goes directly to 100% but if I click on Show log I can see things happening
- [x] If the user pauses the sync and reload the page, the pause/stop should be visible. Clicking on Sync Now will resume it but it is not clear it works like that.

### Errors count
In a site with content imported from WooCommerce sample data and tests/wpa/test-docs/content-example.xml, and also wit a script to cause the `Limit of total fields [5000] in index [...] has been exceeded` error
- [ ] 350 posts per page causes a "Request Entity Too Large" error. That is outputted in the Errors tab but the counter isn't updated
- [x] Full log isn't "full". Errors should be outputted in both places, or we need to change the first tab name
- [ ] Reducing the 350 to 100 I can see 24 lines of errors in that tab but the count says 123

File: includes/partials/sync-page.php (hunk 4 above)
Comment (Rahmon) [reply includes/partials/sync-page.php]: Done

File: includes/partials/sync-page.php (hunk 5 above)
Comment (Rahmon) [reply includes/partials/sync-page.php]: done

File: includes/partials/header.php [hunk 6]
```diff
@@ -23,18 +23,10 @@
...
 			<a class="dashicons cancel-sync dashicons-no" title="<?php esc_attr_e( 'Cancel Sync', 'elasticpress' ); ?>" aria-label="<?php esc_attr_e( 'Cancel Sync', 'elasticpress' ); ?>"></a>
 		<?php endif; ?>
-		<?php if ( Elasticsearch::factory()->get_elasticsearch_version() && defined( 'EP_DASHBOARD_SYNC' ) && EP_DASHBOARD_SYNC ) : ?>
-			<a
-				class="dashicons start-sync dashicons-update"
-				title="<?php esc_attr_e( 'Start Sync', 'elasticpress' ); ?>"
-				aria-label="<?php esc_attr_e( 'Start Sync', 'elasticpress' ); ?>"
-				<?php echo ( $is_sync_page ) ? '' : 'href="' . esc_url( $base_url . 'elasticpress-sync' ) . '"'; ?>
-			></a>
-		<?php endif; ?>
```
[the remark this answers is not in the document]
Comment (Rahmon) [reply includes/partials/header.php]: I brought this button back. It makes sense to me to keep it there.

**Pull request merged**